mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/ElasticSearchWithCassandraMailboxTest.java

```diff
@@ -0,0 +1,146 @@
+import static org.apache.james.mailbox.elasticsearch.MailboxElasticSearchConstants.DEFAULT_MAILBOX_WRITE_ALIAS;
+import static org.apache.james.mailbox.elasticsearch.MailboxIndexCreationUtil.prepareDefaultClient;
+import static org.apache.james.mailbox.events.Event.EventId.random;
+import static org.apache.james.mailbox.store.event.EventFactory.added;
+import static org.apache.james.mailbox.store.mail.MessageMapper.FetchType.Metadata;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.ZoneId;
+
+import org.apache.james.backends.cassandra.CassandraCluster;
+import org.apache.james.backends.cassandra.CassandraClusterExtension;
+import org.apache.james.backends.cassandra.StatementRecorder;
+import org.apache.james.backends.es.DockerElasticSearchExtension;
+import org.apache.james.backends.es.ElasticSearchIndexer;
+import org.apache.james.backends.es.ReactorElasticSearchClient;
+import org.apache.james.core.Username;
+import org.apache.james.mailbox.MailboxSession;
+import org.apache.james.mailbox.MessageManager;
+import org.apache.james.mailbox.cassandra.ids.CassandraId;
+import org.apache.james.mailbox.cassandra.ids.CassandraMessageId;
+import org.apache.james.mailbox.cassandra.mail.MailboxAggregateModule;
+import org.apache.james.mailbox.elasticsearch.IndexAttachments;
+import org.apache.james.mailbox.elasticsearch.MailboxElasticSearchConstants;
+import org.apache.james.mailbox.elasticsearch.MailboxIdRoutingKeyFactory;
+import org.apache.james.mailbox.elasticsearch.MailboxIndexCreationUtil;
+import org.apache.james.mailbox.elasticsearch.events.ElasticSearchListeningMessageSearchIndex;
+import org.apache.james.mailbox.elasticsearch.json.MessageToElasticSearchJson;
+import org.apache.james.mailbox.elasticsearch.query.CriterionConverter;
+import org.apache.james.mailbox.elasticsearch.query.QueryConverter;
+import org.apache.james.mailbox.elasticsearch.search.ElasticSearchSearcher;
+import org.apache.james.mailbox.events.Event;
+import org.apache.james.mailbox.events.Event.EventId;
+import org.apache.james.mailbox.events.MailboxListener;
+import org.apache.james.mailbox.model.Mailbox;
+import org.apache.james.mailbox.model.MailboxPath;
+import org.apache.james.mailbox.model.MessageMetaData;
+import org.apache.james.mailbox.model.MessageRange;
+import org.apache.james.mailbox.store.PreDeletionHooks;
+import org.apache.james.mailbox.store.event.EventFactory;
+import org.apache.james.mailbox.store.extractor.DefaultTextExtractor;
+import org.apache.james.mailbox.store.mail.MessageMapper.FetchType;
+import org.apache.james.metrics.tests.RecordingMetricFactory;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.datastax.driver.core.BoundStatement;
+import com.google.common.collect.ImmutableSet;
+
+public class ElasticSearchWithCassandraMailboxTest {
```

Comment:
This test is not conclusive, maybe we should not merge it?

Reply:
Yes, that why i put it in draft

Reply:
IMO let's remove it, unless you write some tests for the changeset you introduced (update / delete)

Reply:
Could be kept as some sort of a non regression performance test ?
But i am not sure of the value of it